echopype/convert/set_groups_azfp.py

```diff
@@ -124,7 +124,7 @@ def set_sonar(self) -> xr.Dataset:
             "sonar_manufacturer": "ASL Environmental Sciences",
             "sonar_model": self.sonar_model,
             "sonar_serial_number": int(self.parser_obj.unpacked_data["serial_number"]),
-            "sonar_software_name": "Based on AZFP Matlab Toolbox",
+            "sonar_software_name": "AZFP",
```

Comment:
I agree with this change as you proposed in https://github.com/OSOceanAcoustics/echopype/issues/681#issuecomment-1472502461

Reply:
Great, thanks.